test/Elastic.Apm.Feature.Tests/TestConfiguration.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to Elasticsearch B.V under
+// one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Collections.Generic;
+using Elastic.Apm.Config;
+using Elastic.Apm.Helpers;
+using Elastic.Apm.Logging;
+using static Elastic.Apm.Config.ConfigConsts;
+
+namespace Elastic.Apm.Feature.Tests
+{
+	public class TestConfiguration : IConfiguration

```

Comment:
Maybe we can move this into `Elastic.Apm.Tests.Utilities` - as long as we only use it here, it's totally fine at this place as well...

Reply:
I think keeping it here for now is reasonable, and look to refactor `MockConfiguration` to replace this